### PR TITLE
Spelling

### PR DESCRIPTION
**Pull request by jsoref**
This is a followup to #12928.

Whereas most of the corrections in that were made by Google Sheets, this is me manually working through the list of words not in the dictionary...

There's one quasi localized item, `successful` which I dropped from the previous round. I can't find it in Transifex. I think that's because it's a new string and hasn't been uploaded yet. I'm hoping it'd therefore be ok to fix now _before_ it hits Transifex.

### CODE REVIEW

File: database.sql [hunk 1]
```diff
@@ -608,7 +608,7 @@ CREATE TABLE IF NOT EXISTS `diaspora-contact` (
 	`gsid` int unsigned COMMENT 'Global Server ID',
 	`created` datetime NOT NULL DEFAULT '0001-01-01 00:00:00' COMMENT '',
 	`updated` datetime NOT NULL DEFAULT '0001-01-01 00:00:00' COMMENT '',
-	`interacting_count` int unsigned DEFAULT 0 COMMENT 'Number of contacts this contact interactes with',
+	`interacting_count` int unsigned DEFAULT 0 COMMENT 'Number of contacts this contact interacts with',
```
Comment (jsoref) [inline database.sql:611]: I think the previous round also touched things like this, so I think we decided it's ok...

File: doc/Addons.md [hunk 2]
```diff
@@ -264,7 +264,7 @@ $data = [
 ##### With multiple submit buttons
 ```php
 $data = [
-	'addon'  => 'catavar',
+	'addon'  => 'catavatar',
```
Comment (jsoref) [inline doc/Addons.md:267]: I'm pretty sure this is https://git.friendi.ca/friendica/friendica-addons/src/commit/49254a8307f80cfe3ebd165ca556c3acb893b7c6/catavatar

File: src/App/Page.php [hunk 3]
```diff
@@ -253,9 +253,9 @@ private function initHead(App $app, Arguments $args, L10n $l10n, IManageConfigVa
 			'$touch_icon'      => $touch_icon,
 			'$block_public'    => intval($config->get('system', 'block_public')),
 			'$stylesheets'     => $this->stylesheets,
-			'$likeError'       => $l10n->t('Like not successfull'),
-			'$dislikeError'    => $l10n->t('Dislike not successfull'),
-			'$announceError'   => $l10n->t('Sharing not successfull'),
+			'$likeError'       => $l10n->t('Like not successful'),
+			'$dislikeError'    => $l10n->t('Dislike not successful'),
+			'$announceError'   => $l10n->t('Sharing not successful'),
```
Comment (jsoref) [inline src/App/Page.php:258]: As noted, these don't appear to have hit Transifex, so I can't fix them there and no one else could have translated them yet...

File: src/Console/AutomaticInstallation.php [hunk 4]
```diff
@@ -95,7 +95,7 @@ protected function getHelp()
 	bin/console autoinstall --savedb
 		Installs Friendica with environment variables and saves them to the 'config/local.config.php' file
 
-	bin/console autoinstall -H localhost -p 3365 -u user -P passwort1234 -d friendica -U https://friendica.fqdn
+	bin/console autoinstall -H localhost -p 3365 -u user -P password1234 -d friendica -U https://friendica.fqdn
```
Comment (jsoref) [inline src/Console/AutomaticInstallation.php:98]: I'm assuming nothing cares?

File: src/Core/Installer.php [hunk 5]
```diff
@@ -98,17 +98,17 @@ public function __construct()
 	 * Checks the current installation environment. There are optional and mandatory checks.
 	 *
 	 * @param string $baseurl The baseurl of Friendica
-	 * @param string $phpath  Optional path to the PHP binary
+	 * @param string $phppath  Optional path to the PHP binary
```
Comment (jsoref) [inline src/Core/Installer.php]: oops, i guess this is column aligned...

File: src/Module/Api/Twitter/Statuses/Update.php [hunk 6]
```diff
@@ -153,9 +153,9 @@ public function post(array $request = [])
 					continue;
 				}
 
-				Photo::setPermissionForRessource($media[0]['resource-id'], $uid, $item['allow_cid'], $item['allow_gid'], $item['deny_cid'], $item['deny_gid']);
+				Photo::setPermissionForResource($media[0]['resource-id'], $uid, $item['allow_cid'], $item['allow_gid'], $item['deny_cid'], $item['deny_gid']);
 
-				$ressources[] = $media[0]['resource-id'];
+				$resources[] = $media[0]['resource-id'];
```
Comment (jsoref) [inline src/Module/Api/Twitter/Statuses/Update.php]: I did look for column aligned tables, but I guess I missed column aligned source-code...

File: src/Module/Conversation/Network.php [hunk 7]
```diff
@@ -120,7 +120,7 @@ protected function content(array $request = []): string
 			$content = '';
 
 			if (self::$forumContactId) {
-				// If self::$forumContactId belongs to a community forum or a privat goup,.add a mention to the status editor
+				// If self::$forumContactId belongs to a community forum or a private goup,.add a mention to the status editor
```
Comment (jsoref) [inline src/Module/Conversation/Network.php]: The `,.` doesn't look great. 

I'm happy to fix things up if someone provides text (again, my default is to try to only change spelling, as even just the spelling changes are quite a lot to review)

File: src/Protocol/Email.php [hunk 8]
```diff
@@ -51,12 +51,12 @@ public static function connect(string $mailbox, string $username, string $passwo
 
 		$errors = imap_errors();
 		if (!empty($errors)) {
-			Logger::notice('IMAP Errors occured', ['errors' => $errors]);
+			Logger::notice('IMAP Errors occurred', ['errors' => $errors]);
```
Comment (jsoref) [inline src/Protocol/Email.php:54]: This was in the previous PR, but it was in a commit that also changed some localization items which I dropped -- I didn't properly split it out.

File: src/Util/ReversedFileReader.php [hunk 9]
```diff
@@ -158,7 +158,7 @@ public function current(): string
 	}
 
 	/**
-	 * Checks if current iterator value is valid, that is, we readed all lines in files
+	 * Checks if current iterator value is valid, that is, we read all lines in files
```
Comment (jsoref) [inline src/Util/ReversedFileReader.php:161]: I'm not entirely certain about this change

File: src/Worker/ExpirePosts.php [hunk 10]
```diff
@@ -52,7 +52,7 @@ public static function execute()
 			self::addMissingEntries();
 		}
 
-		// Set the expiry for origin posta
+		// Set the expiry for origin post
```
Comment (jsoref) [inline src/Worker/ExpirePosts.php]: ?

File: static/defaults.config.php [hunk 11]
```diff
@@ -520,7 +520,7 @@
 		'profiler' => false,
 
 		// pushpoll_frequency (Integer)
-		// Frequency of contact poll for subhub contact using the DFRM or OStatus network.
+		// Frequency of contact poll for subhub contact using the DFRN or OStatus network.
```
Comment (jsoref) [inline static/defaults.config.php:523]: This _seems_ like a reasonable change, but I'm not particularly confident

File: view/lang/C/messages.po [hunk 12]
```diff
@@ -674,15 +674,15 @@ msgid ""
 msgstr ""
 
 #: src/App/Page.php:256
-msgid "Like not successfull"
+msgid "Like not successful"
```
Comment (jsoref) [inline view/lang/C/messages.po:677]: As noted, these strings haven't arrived in transifex...

File: database.sql (hunk 1 above)
Comment (MrPetovan) [reply database.sql:611]: As long as you also update `static/dbstructure.config.php` and the relevant Markdown documentation page, it's ok!

File: doc/Addons.md (hunk 2 above)
Comment (MrPetovan) [reply doc/Addons.md:267]: Good catch!

File: src/App/Page.php (hunk 3 above)
Comment (MrPetovan) [reply src/App/Page.php:258]: You are right, the import to Transifex is manual, thanks again!

File: src/Console/AutomaticInstallation.php (hunk 4 above)
Comment (MrPetovan) [reply src/Console/AutomaticInstallation.php:98]: No, in this case it is a direct translation from German to English, not necessarily a typo, but it's still warranted. 😄

File: src/Model/Contact.php [hunk 13]
```diff
@@ -106,7 +106,7 @@ class Contact
 	const FOLLOWER = 1; // The contact is following this user (the contact is the subscriber)
 	const SHARING  = 2; // The contact shares their content with this user (the user is the subscriber)
 	const FRIEND   = 3; // There is a mutual relationship between the contact and the user
-	const SELF     = 4; // This is the user theirself
+	const SELF     = 4; // This is the user themselves
```
Comment (MrPetovan) [inline src/Model/Contact.php]: The original word is correct, maybe we should have a Friendica Style Guide? 🤔

File: src/Console/AutomaticInstallation.php (hunk 4 above)
Comment (jsoref) [reply src/Console/AutomaticInstallation.php:98]: A number of the changes in this PR were clearly German equivalents. I think I dropped this item from the previous changeset before I made the original PR (I'm fairly certain Google Sheets suggested it and I decided not to spend the time reasoning through it).

File: src/Module/Api/Mastodon/Statuses.php [hunk 14]
```diff
@@ -391,9 +391,9 @@ private function storeMediaIds(array $media_ids, array $item): array
 				continue;
 			}
 
-			Photo::setPermissionForRessource($media[0]['resource-id'], $item['uid'], $item['allow_cid'], $item['allow_gid'], $item['deny_cid'], $item['deny_gid']);
+			Photo::setPermissionForResource($media[0]['resource-id'], $item['uid'], $item['allow_cid'], $item['allow_gid'], $item['deny_cid'], $item['deny_gid']);
 
-			$ressources[] = $media[0]['resource-id'];
+			$resources[] = $media[0]['resource-id'];
```
Comment (MrPetovan) [inline src/Module/Api/Mastodon/Statuses.php]: This variable isn't used anywhere.
```suggestion
```

File: src/Module/Api/Twitter/Statuses/Update.php [hunk 15]
```diff
@@ -153,11 +153,11 @@ public function post(array $request = [])
 					continue;
 				}
 
-				Photo::setPermissionForRessource($media[0]['resource-id'], $uid, $item['allow_cid'], $item['allow_gid'], $item['deny_cid'], $item['deny_gid']);
+				Photo::setPermissionForResource($media[0]['resource-id'], $uid, $item['allow_cid'], $item['allow_gid'], $item['deny_cid'], $item['deny_gid']);
 
-				$ressources[] = $media[0]['resource-id'];
-				$phototypes   = Images::supportedTypes();
-				$ext          = $phototypes[$media[0]['type']];
+				$resources[] = $media[0]['resource-id'];
```
Comment (MrPetovan) [inline src/Module/Api/Twitter/Statuses/Update.php]: This variable isn't used anywhere.
```suggestion
```

File: src/Module/Conversation/Network.php (hunk 7 above)
Comment (MrPetovan) [inline src/Module/Conversation/Network.php]:
```suggestion
				// If self::$forumContactId belongs to a community forum or a private group, add a mention to the status editor
```

File: src/Protocol/Email.php (hunk 8 above)
Comment (MrPetovan) [reply src/Protocol/Email.php:54]: It's okay, thank you for your work!

File: src/Util/ReversedFileReader.php (hunk 9 above)
Comment (MrPetovan) [reply src/Util/ReversedFileReader.php:161]: Unfortunately, while the fix is correct, it doesn't help much with this counter-intuitive class.

File: src/Worker/ExpirePosts.php (hunk 10 above)
Comment (MrPetovan) [inline src/Worker/ExpirePosts.php]: I believe it's more general.
```suggestion
		// Set the expiry for origin posts
```

File: static/defaults.config.php (hunk 11 above)
Comment (MrPetovan) [reply static/defaults.config.php:523]: Nope, it's good.

File: view/theme/vier/plusminus.css [hunk 16]
```diff
@@ -551,7 +551,7 @@ select{
   border-bottom: 1px solid black;
 }
 
-.setings-contend-block {
+.settings-contend-block {
   background-color: white;
 }
```
Comment (MrPetovan) [inline view/theme/vier/plusminus.css]: Unused anywhere.
```suggestion
```

**Review by MrPetovan — CHANGES_REQUESTED** [review]

File: src/Model/Contact.php (hunk 13 above)
Comment (jsoref) [reply src/Model/Contact.php]: A style guide would definitely help. (In principle, at least localizers and doc writers would want to refer to such an item.)

In my case, I just need to add `theirself` into the local dictionary for this repository.

**Review by MrPetovan — APPROVED** [review]

**Pull request merged**